Solutions/FalconFriday/Analytic Rules/OfficeProcessInjection.yaml

```diff
@@ -0,0 +1,45 @@
+id: a4d8e681-6f30-440a-a2f3-c312bc1389d0
+name: Suspicious Process Injection from Office application
+description: |
+  This query detects process injections.
```

Comment:
please add more meaningful description and recommendation on investigating results